Formula/ptex.rb

```diff
@@ -4,6 +4,7 @@ class Ptex < Formula
   url "https://github.com/wdas/ptex.git",
       tag:      "v2.3.2",
       revision: "1b8bc985a71143317ae9e4969fa08e164da7c2e5"
+  license "Apache-2.0"
```

Comment:
I think this is `BSD-3-Clause`. http://ptex.us/documentation.html#license

Reply:
https://github.com/wdas/ptex/blob/master/Ptex.spec.in#L7 says Apache
https://github.com/wdas/ptex/blob/master/src/doc/License.txt says BSD 🤔

Reply:
> https://github.com/wdas/ptex/blob/master/Ptex.spec.in#L7 says Apache
that is the place I saw.

I think there is license conflict there (but both apache and BSD are permissive licenses though). I will raised in the upstream.

Reply:
That is actually a good catch, I will exclude this formula per the PR.